src/components/SettingDialog.vue

```diff
@@ -272,23 +420,171 @@ export default defineComponent({
         title: "書き出し先のフォルダを選択",
       });
       if (path) {
-        store.dispatch("SET_SAVING_SETTING_DATA", {
+        store.dispatch("SET_SAVING_SETTING", {
           data: { ...savingSetting.value, fixedExportDir: path },
         });
       }
     };
 
-    onUpdated(() => {
-      store.dispatch("GET_SAVING_SETTING_DATA");
+    const hotkeyRows = computed(() => store.state.hotkeySettings);
+
+    const hotkeyColumns = ref([
+      {
+        name: "action",
+        align: "left",
+        label: "操作",
+        field: "action",
+      },
+      {
+        name: "combination",
+        align: "right",
+        label: "ショートカットキー",
+        field: "combination",
+      },
+    ]);
+
+    const lastAction = ref("");
+    const lastRecord = ref("");
+    const lastDuplicated: Ref<HotkeySetting | undefined> = ref(undefined);
```

Comment:
こう書くこともできるはず。
You could write like this.
```suggestion
    const lastDuplicated = ref<HotkeySetting | undefined>(undefined);
```